cmd/root.go

```diff
@@ -67,5 +67,6 @@ func init() {
 
 	if os.Getenv("MENDER_CLI_GEN_AUTOCOMPLETE") != "" {
 		rootCmd.GenBashCompletionFile("./autocomplete/autocomplete.sh")
+		rootCmd.GenZshCompletionFile("./autocomplete/_mender-cli")
```

Comment:
Quite strange naming. Is it better to call this `autocomplete-zsh.sh` or something like that?

Reply:
Hmm, yes. I think we might actually. Or rather, here is the reason:
`Zsh` requires it's autocompletion scripts to have the name `_<tool-name>`. So this will in fact have to be the name once this is put in the correct folder for `zsh` to use. So I guess, weird name here, or weird name later. Which do you prefer?

Reply:
I changed it to `autocomplete.zsh`, and then told the user to change the name upon install. @kacf

Reply:
:+1: